docs/conf.py

```diff
@@ -225,6 +226,249 @@
 
 autodoc_member_order = 'groupwise'
 
+# -- CLI documentation  -------------------------------------------------------
+"""Improves the CLI documentation section.
+
+In order to have separate html pages for each module inside library\\cli
+additional documentation reST files(.rst) need to be generated in docs\\cli folder.
+
+Note:
+    This process assumes that each CLI module represent a group of subcommands.
+    Important: Use hash tables below to address special cases such as
+    command group names different from their module file name.
+"""
+# Activate this CLI documentation process
+custom_cli_docs = True
+# Override existing reSt files found in docs//cli folder
+cli_overwrite = False
+
+# Repository/library hash table.
+# If respository not found in table, the module name will be extracted from the
+# modules.rst file in the project folder.
+# Format: {repository_name: library_name}
+ht_repo_lib = {}
+
+# Library/command hash table.
+# Created to address CLI tool names that differ from their library name beyond
+# the underscore(_) to dash(-) difference.
+# Format: {library_name: tool_name}
+ht_lib_tool = {}
+
+# Cli_file/cli_group hash table.
+# Created to address CLI file names that differ from the included CLI group name
+# Format: {
+#    library_name_1: {CLI_filename_1: group_name_1,
+#                     CLI_filename_2: group_name_2
+#                    },
+#    library_name_2: {CLI_filename_3: group_name_3,
+#                     CLI_filename_4: group_name_4
+#                    }
+# }
+ht_cli_file_group = {
+    'honeybee_radiance': {'mtx': 'mtxop',
+                          'postprocess': 'post_process'
+                          }
+}
```

Comment:
Given that this repo is does not have these `honeybee-radiance`  modules (or any modules that have a different name than the CLI group), could we remove this part?  If you want to leave the comments in case we add modules with different group names in the future, I think that's fine. But it doesn't seem right to pollute this repo with the baggage of honeybee-radiance. It's a totally separate reop, after all, and it runs independently of honeybee-radiance.